packages/arb-validator/cmdhelper/cmdhelper.go

```diff
@@ -21,115 +21,51 @@ import (
 	"github.com/offchainlabs/arbitrum/packages/arb-validator/rollupmanager"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator/rollupvalidator"
-
-	"golang.org/x/crypto/ssh/terminal"
-
-	"github.com/ethereum/go-ethereum/accounts"
-	"github.com/ethereum/go-ethereum/accounts/abi/bind"
-	"github.com/ethereum/go-ethereum/accounts/keystore"
 )
 
-func GetKeystore(validatorFolder string, pass *string, flags *flag.FlagSet) (*bind.TransactOpts, error) {
-	ks := keystore.NewKeyStore(filepath.Join(validatorFolder, "wallets"), keystore.StandardScryptN, keystore.StandardScryptP)
-
-	found := false
-	flags.Visit(func(f *flag.Flag) {
-		if f.Name == "password" {
-			found = true
-		}
-	})
-
-	var passphrase string
-	if !found {
-		if len(ks.Accounts()) == 0 {
-			fmt.Print("Enter new account password: ")
-		} else {
-			fmt.Print("Enter account password: ")
-		}
-
-		bytePassword, err := terminal.ReadPassword(int(syscall.Stdin))
-		if err != nil {
-			return nil, err
-		}
-		passphrase = string(bytePassword)
-
-		passphrase = strings.TrimSpace(passphrase)
-	} else {
-		passphrase = *pass
-	}
-
-	var account accounts.Account
-	if len(ks.Accounts()) == 0 {
-		var err error
-		account, err = ks.NewAccount(passphrase)
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		account = ks.Accounts()[0]
-	}
-	err := ks.Unlock(account, passphrase)
-	if err != nil {
-		return nil, err
-	}
-	auth, err := bind.NewKeyStoreTransactor(ks, account)
-	if err != nil {
-		return nil, err
-	}
-	return auth, nil
-}
-
 func ValidateRollupChain(execName string, managerCreationFunc func(rollupAddress common.Address, client arbbridge.ArbAuthClient, contractFile string, dbPath string) (*rollupmanager.Manager, error)) error {
 	// Check number of args
 
 	validateCmd := flag.NewFlagSet("validate", flag.ExitOnError)
-	passphrase := validateCmd.String("password", "", "password=pass")
+	walletVars := utils.AddFlags(validateCmd)
 	rpcEnable := validateCmd.Bool("rpc", false, "rpc")
 	blocktime := validateCmd.Int64("blocktime", 2, "blocktime=NumSeconds")
-	gasPrice := validateCmd.Float64("gasprice", 4.5, "gasprice=FloatInGwei")
 	err := validateCmd.Parse(os.Args[2:])
 	if err != nil {
 		return err
 	}
 
 	if validateCmd.NArg() != 3 {
-		return fmt.Errorf("usage: %v validate [--password=pass] [--rpc] [--blocktime=NumSeconds] [--gasprice==FloatInGwei] <validator_folder> <ethURL> <rollup_address>", execName)
+		return fmt.Errorf("usage: %v validate %v [--rpc] [--blocktime=NumSeconds] %v", execName, utils.WalletArgsString, utils.RollupArgsString)
 	}
 
 	common.SetDurationPerBlock(time.Duration(*blocktime) * time.Second)
 
-	validatorFolder := validateCmd.Arg(0)
-	ethURL := validateCmd.Arg(1)
-	addressString := validateCmd.Arg(2)
-	address := common.HexToAddress(addressString)
+	rollupArgs := utils.ParseRollupCommand(validateCmd, 0)
 
-	auth, err := GetKeystore(validatorFolder, passphrase, validateCmd)
+	auth, err := utils.GetKeystore(rollupArgs.ValidatorFolder, walletVars, validateCmd)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 87 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)